### PR TITLE
docs(document-driven): fix typo

### PR DESCRIPTION
**Pull request by therealokoro**
Fixed typo in line 114 `Queries are be made` to `Queries are been made`

### CODE REVIEW

File: docs/content/3.guide/1.writing/7.document-driven.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ This option will search for a `theme` key in `globals`, then search for a `layou
 
 ## Global variables
 
-Queries are be made from a [route middlewares](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
+Queries are been made from a [route middlewares](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
```
Comment (nobkd) [inline docs/content/3.guide/1.writing/7.document-driven.md:114]:
```suggestion
Queries are made from a [route middleware](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
```

Comment (nobkd) [inline docs/content/3.guide/1.writing/7.document-driven.md:114]:
```suggestion
Queries are made from a [route middleware](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
```

**Review by nobkd — CHANGES_REQUESTED** [review]

File: docs/content/3.guide/1.writing/7.document-driven.md (hunk 1 above)
Comment (therealokoro) [reply docs/content/3.guide/1.writing/7.document-driven.md:114]: much better. Thanks... I made a mess of correction. It should be `being` not `been`.

**Review by therealokoro — COMMENTED** [review]
mistakes upon mistakes 🤪

**Comment by nobkd** [comment]
Ah. Consider changing the title of the PR to follow the [conventional commits syntax](https://www.conventionalcommits.org/en/v1.0.0/)
e.g. `docs(document-driven): fix typo`

**Review by farnabaz — APPROVED** [review]

**Pull request merged**